src/core/StickyContainer.tsx

```diff
@@ -99,12 +101,27 @@ export default class StickyContainer<P extends StickyContainerProps> extends Com
                         getRowRenderer={this._getRowRenderer}
                         overrideRowRenderer={this.props.overrideRowRenderer}
                         renderContainer={this.props.renderStickyContainer}
-                        getWindowCorrection={this._getCurrentWindowCorrection} />
+                        getWindowCorrection={this._getCurrentWindowCorrection}
+                        alwaysStickBottom = {this.props.alwaysStickBottom} />
                 ) : null}
             </View>
         );
     }
 
+    private _rlvRowRenderer = (type: string | number, data: any, index: number, extendedState?: object): JSX.Element | JSX.Element[] | null => {
+        const rlvDimension: Dimension | undefined = this._getRLVRenderedSize();
+        const contentDimension: Dimension | undefined = this._getContentDimension();
+        let isScrollable = false;
+        if (rlvDimension && contentDimension) {
+            isScrollable = contentDimension.height > rlvDimension.height;
+        }
+        if (this.props.alwaysStickBottom && !isScrollable && this.props.stickyFooterIndices
+            && index === this.props.stickyFooterIndices[0]) {
+            return null;
+        }
+        return this._rowRenderer(type, data, index, extendedState);
```

Comment:
Let's have an outer check of `this.props.alwaysStickBottom`, `this.props.stickyFooterIndices` and `index === this.props.stickyFooterIndices[0]` before calculating isScrollable. Return the usual otherwise.

Reply:
done